examples/node_tree.py

```diff
@@ -10,20 +10,16 @@
 
 """ Node tree example. """
 
+import os
 
-import os, sys
-
-# Put the Enthought library on the Python path.
-sys.path.append(os.path.abspath(r"..\..\.."))
-
-
-from pyface.api import GUI, PythonShell, SplitApplicationWindow
```

Comment:
feel free to ignore - I think this import can stay where it is - the examples live outside the pyface package so can treat the pyface import as just a part of the ETS import group.

Reply:
True, but I'll leave it for now and add an issue, because I suspect that there are more cases.